src/Cache/Cache.php

```diff
@@ -500,6 +632,19 @@ public static function clear($check = false, $config = 'default')
         return $engine->clear($check);
     }
 
+    /**
+     * Clears only expired cache entries
+     *
+     * @param string $config name of the configuration to use. Defaults to 'default'
+     * @return bool
+     */
+    public function clearExpired($config = 'default')
```

Comment:
If this isn't part of the PSR interfaces why are we adding it? It won't do anything on most engines as they auto-expire data.